### DISCUSSION TITLE
Does the model analyse speaker.wav every time it runs?

### DISCUSSION THREAD

**Discussion by killswitchh** [coqui-ai/TTS#3265] [General Q&A]
`tts_to_file(text=prompt,
                file_path="output_audio.wav",
                speaker_wav="speaker.wav",
                language="en")`
I have tts_to_file at multiple places in my code.
I want to try out a method in which the model analyses speaker.wav once so that it can take less time during the second call.

**Comment by erogol** [comment]
for most models, yes. Only bark caches the speaker.
